src/containers/home/contact/component.tsx

```diff
@@ -63,117 +59,145 @@ const Contact: React.FC = () => {
+                        component="input"
+                        validate={composeValidators([{ presence: true, email: true }])}
+                      >
+                        {({ input }) => (
+                          <motion.div
+                            className="relative w-full space-y-2"
+                            animate={{ opacity }}
+                            transition={{ delay: 0.4 }}
+                          >
+                            <label className="pl-4 text-lg font-semibold text-gray-100">
+                              Email
+                            </label>
+                            <input
+                              {...input}
+                              value={input.value as string}
+                              type="email"
+                              className="flex w-full h-16 p-4 text-base text-gray-400 transition duration-300 ease-in-out delay-150 bg-gray-900 border-gray-800 rounded-lg focus:outline-none focus:ring-inset focus:ring-2 focus:ring-brand-700 md:text-lg md:py-5"
+                            />
+                          </motion.div>
+                        )}
+                      </Field>
+
+                      <Field name="interests" component="input">
+                        {({ input }) => (
+                          <motion.div
+                            className="relative w-full space-y-2"
+                            animate={{ opacity }}
+                            transition={{ delay: 0.4 }}
+                          >
+                            <label className="pl-4 text-lg font-semibold text-gray-100">
+                              I am interested in
+                            </label>
+                            <select
+                              {...input}
+                              className="block w-full h-16 px-4 py-4 m-0 font-sans text-lg text-gray-400 transition ease-in-out bg-gray-800 bg-no-repeat border border-gray-800 rounded-lg appearance-none form-select bg-clip-padding focus:outline-none focus:ring-inset focus:ring-2 focus:ring-brand-700"
+                            >
+                              <option selected>{''}</option>
+                              {INTERESTS.map(({ label, value }) => {
+                                return (
+                                  <option key={value} value={value}>
+                                    {label}
+                                  </option>
+                                );
+                              })}
+                            </select>
+                          </motion.div>
+                        )}
+                      </Field>
+
+                      <motion.div
+                        animate={{ opacity }}
+                        transition={{ delay: 0.5, bounce: 0 }}
+                        className="w-full"
+                      >
+                        <Button
+                          disabled={!form.getState().valid}
```

Comment:
Why did you remove the submitting state? You should add it to this condition. Users shouldn't be able to submit the form while the form is submitting